examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -142,8 +190,75 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 					`Registered for webhook notifications at URL: "${subscriberUrl}".`,
 				),
 			);
-			result.send();
+		} catch (error) {
+			if (error instanceof ApiError) {
+				console.warn(formatLogMessage(error.message));
```

Comment:
Nit: should this be `console.error`?

Reply:
Updated to be warn or error based on if the error code is >= 500